tests/integration/spark/test_spark_config.py

```diff
@@ -45,56 +45,39 @@ def test_spark_config_datasource(spark_session_v012):
     assert ("spark.app.name", name) in conf
     assert ("spark.sql.catalogImplementation", "hive") in conf
     assert ("spark.executor.memory", "768m") in conf
+    spark_session.sparkContext.stop()
 
 
-@pytest.mark.parametrize("databricks_runtime", [False, True])
-def test_spark_config_execution_engine(spark_session, databricks_runtime):
-    # we observe different behaviour depending on Spark version:
-    # https://spark.apache.org/docs/3.0.0/pyspark-migration-guide.html#upgrading-from-pyspark-24-to-30
-    pyspark_version = parse_version(pyspark.__version__)
+def test_spark_config_execution_engine(spark_session):
+    pyspark_version: Version = parse_version(pyspark.__version__)
+    # Spark versions less than 3.0 are not supported.
+    assert pyspark_version.major >= 3
 
```

Comment:
How about putting this in a separate test? Also is this necessary to test? We may want to assert at runtime (e.g. to raise a helpful error if someone is running with spark <3.0). But otherwise we should make sure our test config installs >= 3.0

Reply:
@anthonyburdi I will do that.

Reply:
@anthonyburdi Committed